packages/filebrowser/src/listing.ts

```diff
@@ -1189,12 +1191,42 @@ export class DirListing extends Widget {
       proposedAction: 'move'
     });
     let basePath = this._model.path;
+
     let paths = toArray(
       map(selectedNames, name => {
         return PathExt.join(basePath, name);
       })
     );
     this._drag.mimeData.setData(CONTENTS_MIME, paths);
+
+    let services = this.model.manager.services;
+    const selectedItems: Contents.IModel[] = [];
+    Object.keys(this._selection).forEach(itemname => {
+      if (this._selection[itemname]) {
+        const item = find(items, item => item.name === itemname);
+        if (
+          item.type === 'file' &&
+          imageRendererFactory.mimeTypes.indexOf(item.mimetype) !== -1
+        ) {
+          selectedItems.push(item);
```

Comment:
This entire logic can be removed. Instead, it should only check `item.mimetype !== null` (implies file type), and add a thunk for it. Then the drop target will check the types against the imageRendererFactory. That will allow us to remove the import from rendermime in this pacakge.

Reply:
Yes, that is much simpler! Removed this logic now